x-pack/test/functional_with_es_ssl/apps/triggers_actions_ui/connectors/utils.ts

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type SuperTest from 'supertest';
+import { findIndex } from 'lodash';
+
+import { ObjectRemover } from '../../../lib/object_remover';
+import { FtrProviderContext } from '../../../ftr_provider_context';
+import { getTestActionData } from '../../../lib/get_test_data';
+
+export const createConnectorAndObjectRemover = async ({
```

Comment:
nit: As it is not generic and creates a slack connector maybe is better to rename it to `createSlackConnectorAndObjectRemover`

Reply:
Great point.